test/suggestion_engine/server/index.test.js

```diff
@@ -63,6 +63,24 @@ test('should call appropriate activation methods', async () => {
   expect(browser.tabs.update.calledOnce).toEqual(true);
 });
 
+test('should focus bookmark and history tabs if already open', async () => {
+  browser.flush();
+  browser.tabs.query.returns(Promise.resolve([{ id: '1', windowId: '1' }]));
+  await activateSuggestion({
```

Comment:
Can the tests in the previous test ('should call appropriate activation methods') for the bookmark and history mode be deleted since we have coverage from these?

Reply:
The previous tests ensure that the "create a new tab" behaviour still occurs when no existing tabs are open. It's a little unclear that this is the case though. I didn't want to change the file too much as it's my first PR to this repo.